gui.py

```diff
@@ -0,0 +1,82 @@
+from conversion_manager import ConversionManager
+from path_helper import PathHelper
+from PyQt6.QtCore import QSize
+from PyQt6.QtGui import QIcon
+from PyQt6.QtWidgets import QApplication, QMainWindow, QPushButton, QVBoxLayout, QHBoxLayout, QLabel, \
+    QWidget, QFileDialog, QMessageBox
+
+
+class MainWindow(QMainWindow):
+    def __init__(self, conversion_manager: ConversionManager, input_path: str, output_path: str) -> None:
+        super().__init__()
+        self.setMinimumSize(QSize(300, 150))
+
+        self.conversion_manager: ConversionManager = conversion_manager
+        self.input_path: str = input_path
+        self.output_path: str = output_path
+
+        self.setWindowTitle("Аршин-конвертер")
+        self.layout = QVBoxLayout()
+        button_layout = QHBoxLayout()
+
+        search_button = QPushButton("Обзор")
+        start_button = QPushButton("Начать")
+        search_button.clicked.connect(self.__the_search_button_was_clicked)
+        start_button.clicked.connect(self.__the_start_button_was_clicked)
+
+        if self.input_path:
+            self.input_path = PathHelper.get_abspath(self.input_path)
```

Comment:
What does this method exactly do?
I'm not sure it's getting a path, but transforms/resolves it from something else. Depending on that the name may be different.